lib/MusicBrainz/Server/Email.pm

```diff
@@ -452,9 +415,39 @@ sub send_message_to_editor
 sub send_email_verification
 {
     my ($self, %opts) = @_;
+    my $_url = $mail_service_base_url . "/send_single";
 
-    my $email = $self->_create_email_verification_email(%opts);
-    return $self->_send_email($email);
+    my $ip = $opts{ip};
+    my $to_name = $opts{editor}->name;
+    my $verification_link = $opts{verification_link};
+
+    if(blessed($verification_link) && $verification_link->can('as_string')) {
+        $verification_link = $verification_link->as_string;
+    }
+
+    my $body = {
+        'template_id' => 'verify-email',
+        'to'          => $opts{email},
+        'from'        => $EMAIL_NOREPLY_ADDRESS,
+        # 'lang'
+        'message_id'  => _message_id('verify-email-%d', time()),
+        'reply_to'  => $EMAIL_NOREPLY_ADDRESS,
+        'params'      => {
+            'to_name'          => $to_name,
+            'verification_url' => "$verification_link",
```

Comment:
If this is already a string, why stringify it again?

FWIW, at least locally the string does not seem to be empty.

Reply:
That issue seems to have resolved itself. I believe it was because it was because bitmap and I were testing with an older version of the Mail service image 🤷. The reason for the two methods of stringification was effectively me throwing stuff at the wall to see what stuck - I can probably remove the first one now.